src/main/java/uk/gov/companieshouse/confirmationstatementapi/service/FilingService.java

```diff
@@ -0,0 +1,66 @@
+package uk.gov.companieshouse.confirmationstatementapi.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.api.model.filinggenerator.FilingApi;
+import uk.gov.companieshouse.confirmationstatementapi.exception.SubmissionNotFoundException;
+import uk.gov.companieshouse.confirmationstatementapi.model.json.ConfirmationStatementSubmissionJson;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+@Service
+public class FilingService {
+
+    @Value("${CONFIRMATION_STATEMENT_NO_UPDATES}")
```

Comment:
the property name doesn't indicate it is a description, it makes it sound like. boolean.
I recommend CONFIRMATION_STATEMENT_DESCRIPTION_NO_UPDATES